src/setup/cortx_setup.py

```diff
@@ -72,23 +95,27 @@ def process(self, *args, **kwargs):
 
 
 def main():
+    log_path = os.path.join('/var/log/cortx', 'provisioner')
+    Log.init('cortx_setup', log_path, console_output=True)
+
     try:
         # Parse and Process Arguments
         command = Cmd.get_command(sys.modules[__name__], 'cortx_setup', sys.argv[1:])
+        command.validate()
         rc = command.process()
 
     except CortxProvisionerError as e:
-        sys.stderr.write("%s\n\n" % str(e))
-        sys.stderr.write("%s\n" % traceback.format_exc())
+        Log.error('%s' % str(e))
```

Comment:
Ensure that this is also printed on the stderr for non K8s env.

Reply:
Okay